pkg/metricscollector/v1beta1/common/const.go

```diff
@@ -40,7 +40,10 @@ const (
 	// Metrics must be printed this way
 	// loss=0.3
 	// accuracy=0.98
-	DefaultFilter = `([\w|-]+)\s*=\s*((-?\d+)(\.\d+)?)`
+	// Score=-7.53e-05
```

Comment:
Please can you also add the following example, I believe it is supported.
```suggestion
	// Score=-7.53e-05
	// Score=-7.53e+05
```